modules/search/pages/run-searches.adoc

```diff
@@ -48,6 +77,57 @@ For more information about how to configure a Search index and search for geospa
 
 Use the xref:clusters:query-service/query-workbench.adoc[Query tab] to search using natural-language search and {sqlpp} features in the same query. 
 
+When using {sqlpp} with a hybrid xref:vector-search:vector-search.adoc[Vector Search] query, you have more flexibility in how you choose to display your search results. 
+When running a hybrid search with the <<ui,Web Console>> or <<api,REST API>>, the Search Service displays results as a disjunct (`OR`) between your 2 search queries. 
+For example: 
+
+----
+
+{ 
+    "query": 
+    {
+        "match_phrase": "my regular query"
+    } 
+}
+
+OR 
+
+{
+    "knn": [
+        "k": 5,
+        "field": "vector_field",
+        "vector": [0, 0, 128]
+    ]
+}
+----
+
+{sqlpp} allows you to choose whether to return search results as a conjunct (`AND`) or a disjunct (`OR`) between for hybrid search queries.
+
+As a conjunct, the Search Service returns matches that score highly for both the regular Search query and the Vector Search query. 
+The Search Service would exclude matches that only match the Vector Search query.
```

Comment:
This is totally just a suggestion and feel free to ignore, but what about bullet points here? Like:

As a conjunct, the Search Service:
* Returns matches that score highly for both the regular Search query and the Vector Search query.
* Excludes matches that only match the Vector Search query.